storage/indexer.go

```diff
@@ -213,13 +217,16 @@ func (i *Indexer) Get(ctx context.Context, opts *packages.GetOptions) (packages.
 	return i.packageList, nil
 }
 
-func (i *Indexer) transformSearchIndexAllToPackages(sia searchIndexAll) (packages.Packages, error) {
+func (i *Indexer) transformSearchIndexAllToPackages(sia *searchIndexAll) (packages.Packages, error) {
```

Comment:
Maybe we can get rid of the `searchIndexAll` struct, see this patch over your changes: https://github.com/jsoriano/package-registry/commit/6875d6f1d335825f453859624dec1cecb04da323